src/main/java/duke/command/DueInCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+
+public class DueInCommand extends Command {
+    private long time;
```

Comment:
I am not sure what this variable is for. Similarly for `hour`. Would it be better to rename this variable to be more descriptive?